waku/node.go

```diff
@@ -231,7 +231,7 @@ func Execute(options Options) {
 		options.Relay.Topics = []string{string(relay.DefaultWakuTopic)}
 	}
 
-	if !options.Relay.Disable {
+	if wakuNode.CanSubscribeToTopics() {
 		for _, nodeTopic := range options.Relay.Topics {
 			_, err := wakuNode.Relay().SubscribeToTopic(ctx, nodeTopic)
```

Comment:
Something like this will work. It will still create the pubsub subscription, but it won't leave you with an unused broadcaster subscription.
```go
sub, err := wakuNode.Relay().SubscribeToTopic(ctx, nodeTopic)
failOnErr(err, "Error subscring to topic")
wakuNode.Broadcaster().Unregister(sub.C)
```